Starting with glib 2.32 it is now mandatory to include glib.h instead
of individual headers [1], or the compiler will generate an error.

With the notable exception of:
 glib/gi18n.h
 glib/gi18n-lib.h
 glib/gprintf.h
 glib/gstdio.h
Those header files are still meant to be included separately.
Index: minbif-1.0.5/src/server_poll/daemon_fork.cpp
===================================================================
--- minbif-1.0.5.orig/src/server_poll/daemon_fork.cpp	2012-03-27 11:10:49.000000000 +0200
+++ minbif-1.0.5/src/server_poll/daemon_fork.cpp	2012-03-27 11:19:18.000000000 +0200
@@ -20,7 +20,7 @@
 #include <cassert>
 #include <cstring>
 #include <cerrno>
-#include <glib/gmain.h>
+#include <glib.h>
 #include <sys/socket.h>
 #include <sys/stat.h>
 #include <arpa/inet.h>
Index: minbif-1.0.5/src/server_poll/inetd.cpp
===================================================================
--- minbif-1.0.5.orig/src/server_poll/inetd.cpp	2012-03-27 11:10:43.000000000 +0200
+++ minbif-1.0.5/src/server_poll/inetd.cpp	2012-03-27 11:19:13.000000000 +0200
@@ -17,7 +17,7 @@
  */
 
 #include <cassert>
-#include <glib/gmain.h>
+#include <glib.h>
 
 #include "inetd.h"
 #include "irc/irc.h"
